src/Umbraco.Web.UI/Umbraco.Web.UI.csproj

```diff
@@ -837,6 +837,18 @@
     <Content Include="Views\Partials\Grid\Editors\Base.cshtml" />
     <Content Include="Views\Partials\Grid\Bootstrap3-Fluid.cshtml" />
     <Content Include="Views\Partials\Grid\Bootstrap2-Fluid.cshtml" />
+    <Content Include="Umbraco\Views\Preview\Background.cshtml" />

```

Comment:
Did these files already exist @clausjensen 
Or if they are brand new - then they are missing from this PR